src/ckb/channel.rs

```diff
@@ -145,13 +148,20 @@ pub enum ChannelInitializationParameter {
     /// To open a new channel to another peer, the funding amount,
     /// the temporary channel id a unique channel seed to generate
     /// channel secrets must be given.
-    OpenChannel(u128, [u8; 32], Option<Script>, oneshot::Sender<Hash256>),
+    OpenChannel(
```

Comment:
Better using a named struct instead of tuple.

Reply:
fixed.